tests/test_uflash.py

```diff
@@ -300,10 +311,10 @@ def test_flash_wrong_python():
     Ensures a call to flash will fail if it's not reported that we're using
     Python 3.
     """
-    with mock.patch('sys.version_info', (2, 7, 9)):
-        with pytest.raises(RuntimeError) as ex:
-            uflash.flash()
-    assert ex.value.args[0] == 'Will only run on Python 3.3 or later.'
+    for version in [(2, 6, 3), (3, 2, 0)]:
+        with mock.patch('sys.version_info', version):
+            with pytest.raises(RuntimeError):
+                uflash.flash()
```

Comment:
OK... I remember why I tested the error string. It was to ensure that there was a helpful message since many of our users will be kids / teacher / new programmers. By testing for it, should it ever get removed would ensure that we have to think about how we communicate the wrong version of Python to the end user. It worked!!! :-)

Given our likely user's lack of experience, can we keep this assertion in (for the reasons described above). The new message should be: "This program only works with Python version 2.7 or Python version 3.3 or later."

Cheers..! :smile: